docs/api/ref/api-v3.yml

```
@@ -38,6 +38,7 @@ paths:
           schema:
             type: string
             example: "3017620422003"
+        - $ref: "#/components/parameters/product_type"
```

why not directly:
```suggestion
        - $ref: "./parameters/requested_product_type.yaml#/components/parameters/RequestedProductType"
```
?

---

Sounds good, I'll use  the direct reference then. Thank you!